app/models/sushi.rb

```diff
@@ -112,6 +112,23 @@ def rescued_last_month_available(*args)
     end
   end
 
+  ##
+  # This module provides a common interface for validating parameters.
+  #
+  # @param params [Hash, ActionController::Parameters]
+  # @param allowed_parameters [Array<String>] a list of query parameters AND report filters that are allowed.
```

Comment:
@jeremyf asking for your new review due to this change. it also required a change in the specs.

Reply:
It looks good.  The specs are good to update.